docs: Updated docs following up on repo transfer

frgfm opened the pull request:
This PR updates URLs following up on repo transfer by:
- updating badges
- updating web page references
- updating repo URL references

frgfm committed:
docs: Updated URLs of contributing

frgfm committed:
docs: Updated URLs in README

x0s reviewed (changes requested):
Thanks the PRs and having transferred the repo to the organization.

Could you also update the [setup](https://github.com/pyronear/PyroNear/blob/master/setup.py#L49-L50) nd the [bug template](https://github.com/pyronear/PyroNear/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) ?

frgfm committed:
chore: Updated setup URLs

frgfm committed:
docs: Updated bud report template

frgfm committed:
docs: Removed unused URL brackets

frgfm committed:
fix: Resolved conflicts on README

The same line was modified on a PR that was merged after this one was opened. Merged master into this to avoid any issue

frgfm committed:
docs: Fixed README URL merge typo

In resolving the conflict, a typo was introduced in the sphinx doc URL.

frgfm commented:
> Thanks the PRs and having transferred the repo to the organization.
> 
> Could you also update the [setup](https://github.com/pyronear/PyroNear/blob/master/setup.py#L49-L50) nd the [bug template](https://github.com/pyronear/PyroNear/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) ?

Good catch thanks!
I just updated them (fwi I had to merge master into this branch to resolve a conflict due to modifications on both branches of the same line in README) :ok_hand:

x0s reviewed (approved):
Looks good to me!
